aten/src/ATen/native/native_functions.yaml

```diff
@@ -4574,6 +4579,8 @@
   device_check: NoCheck
   device_guard: False
 
+- func: efficientzerotensor(int[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
```

Comment:
I hate this function name lol

Reply:
Alban and I were thinking of adding a kwarg argument to `torch.zeros` and expose it to users in that way. This is only a temp function to get the basic stuff working